deeployer.libsonnet

```diff
@@ -62,6 +62,7 @@
               "API_URL": "back1:50051,back2:50051",
               "SECRET_JITSI_KEY": env.SECRET_JITSI_KEY,
               "FRONT_URL": "https://play-"+url,
+              "ENABLE_OPENAPI_ENDPOINT": "true",
```

Comment:
I don't see where is used this environment variable?

Reply:
Here: http://github.com/thecodingmachine/workadventure/blob/b152f1ce108365df16dfbd2f753f02bd552a980c/pusher/src/App.ts#L36-L38